test/cases/regression-composer-works-behind-satellite.sh

```diff
@@ -65,17 +65,19 @@ case "${ID}" in
         echo "Running on RHEL"
         case "${VERSION_ID%.*}" in
             "8" )
-                echo "Running on RHEL 8"
+                echo "Running on RHEL ${VERSION_ID}"
+                # starting in 8.5 the override file contains minor version number as well
+                VERSION_SUFFIX=$(echo "${VERSION_ID}" | tr -d ".")
                 if grep beta /etc/os-release;
                 then
                     DISTRO_NAME="rhel-8-beta"
-                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-8-beta.json"
+                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-${VERSION_SUFFIX}-beta.json"
                 else
                     DISTRO_NAME="rhel-8"
-                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-8.json"
+                    REPOSITORY_OVERRIDE="/etc/osbuild-composer/repositories/rhel-${VERSION_SUFFIX}.json"
                 fi
-                REPO1_NAME="baseos"
-                REPO2_NAME="appstream"
+                REPO1_NAME="baseos-${ARCH}"
+                REPO2_NAME="appstream-${ARCH}"
```

Comment:
Won't this break regular pipelines? I think you renamed the repos only for the nightly pipeline...

Reply:
Duh. We need more conditionals here :-(